aksetup_helper.py

```diff
@@ -956,6 +953,10 @@ def build_extensions(self):
             cxx_opts.append(cpp_flag(self.compiler))
             if has_flag(self.compiler, '-fvisibility=hidden'):
                 opts.append('-fvisibility=hidden')
+            if has_flag(self.compiler, '-stdlib=libc++'):
+                opts.append('-stdlib=libc++')
+            if has_flag(self.compiler, '-mmacosx-version-min=10.7'):
+                opts.append('-mmacosx-version-min=10.7')
```

Comment:
This will attempt to add these flags even on non-Darwin platforms, which I don't think is right.

Reply:
Oops. 🙂  Fixed.